### PR TITLE
fix(component):Add AGE property for workload

### LINKED ISSUE #96: Add AGE property for workload
It's a little bit difficult for users to quickly identify the component he/she created some time ago.
```
k get component
NAME       WORKLOAD-KIND
app-0709   ContainerizedWorkload
f0709      ContainerizedWorkload
frontend   ContainerizedWorkload
poc        ContainerizedWorkload
```

It would be better if `AGE` property can also be shown as `ApplicationConfiguration`.
```
k get applicationconfiguration
NAME       AGE
app-0709   32h
f0709      21h
frontend   4d20h
poc        4d20h
```

### PR DESCRIPTION
**Pull request by Ghostbaby**
Add AGE property for workload

Fixes #96

Signed-off-by: zhuhuijun <zhuhuijunzhj@gmail.com>

### CODE REVIEW

File: apis/core/v1alpha2/core_types.go [hunk 1]
```diff
@@ -221,6 +221,7 @@ type Revision struct {
 // +kubebuilder:resource:categories={crossplane,oam}
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:JSONPath=".spec.workload.kind",name=WORKLOAD-KIND,type=string
+// +kubebuilder:printcolumn:name="age",type="date",JSONPath=".metadata.creationTimestamp"
```
Comment (ryanzhang-oss) [inline apis/core/v1alpha2/core_types.go:224]: is the timestamp same as AGE (which is the difference between NOW and the creationtime"

File: charts/oam-core-runtime/crds/core.oam.dev_components.yaml [hunk 2]
```diff
@@ -12,6 +12,9 @@ spec:
   - JSONPath: .spec.workload.kind
     name: WORKLOAD-KIND
     type: string
+  - JSONPath: .metadata.creationTimestamp
+    name: age
+    type: date
```
Comment (ryanzhang-oss) [inline charts/oam-core-runtime/crds/core.oam.dev_components.yaml:17]: does this convert timestamp to (now-timestamp)?

Comment (wonderflow) [reply charts/oam-core-runtime/crds/core.oam.dev_components.yaml]: yes, it's mechanism of CRD

File: apis/core/v1alpha2/core_types.go (hunk 1 above)
Comment (wonderflow) [reply apis/core/v1alpha2/core_types.go:224]: It's create time, evaluation will automatically done by k8s

**Pull request merged**